app/models/container_route.rb

```diff
@@ -3,5 +3,6 @@ class ContainerRoute < ActiveRecord::Base
   include ReportableMixin
 
   belongs_to :ext_management_system, :foreign_key => "ems_id"
-  has_many :labels, -> { where(:section => "labels") }, :class_name => "CustomAttribute", :as => :resource
+  belongs_to :container_services
```

Comment:
My comment is still valid this has to be:

    belongs_to :container_service

(singular)